Cargo.toml

```diff
@@ -55,9 +55,35 @@ serde_repr = { version = "0.1", optional = true }
 [dev-dependencies]
 pretty_assertions = "0.5.1"
 assert_matches = "1.4.0"
+criterion = "0.3"
+pprof = { version = "0.4.4", features = ["criterion", "flamegraph"] }
+utilities = {path = "utilities"}
 
 [workspace]
 members = [
     ".",
     "lsp/nls"
 ]
+
+[profile.release]
+debug = true
```

Comment:
Do we really want to include debugging symbols in the release profile? What is this for?

Reply:
I commented it out, and added a comment that says `# Enable this to use flamegraphs`.